app/views/users/show.html.erb

```diff
@@ -24,6 +24,9 @@
         <%= link_to "See all posts", user_posts_path(@user), {:class => "link_class"} %>
```

Comment:
[OPTIONAL] On the users show page, it would be better if the user is able to navigate to the post page by clicking on the post title to improve the user experience.